plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/internal/SQLEditorMessages.properties

```diff
@@ -346,5 +347,15 @@ sql_generator_title_text = Generated SQL ( {0} )
 sql_generator_no_obj_container_text = Unable to get container
 sql_generator_no_ddl_text = No DDL is available for the result set
 sql_generator_nonsql_text = The container does not support the standard DDL structure.
-
 sql_editor_separate_connection_no_editor_or_ds_selected = No editor or data source selected
+
+sql_editor_prefs_disable_services_text = Disable SQL syntax parser
+sql_editor_prefs_disable_services_tip = Overrides settings for code completion, foldings and word occurrences highlighting\n(Automatically applied for large script files)
+
+sql_editor_prefs_script_advanced_settings = Advanced settings
+sql_editor_prefs_script_disable_sql_syntax_parsing_for_scripts_bigger_than = Disable SQL syntax parser for files bigger than (KB)
+sql_editor_confirm_no_fetch_result_for_big_script_title = Preparing to execute the script
+sql_editor_confirm_no_fetch_result_for_big_script_question = Large script execution may be dramatically slow while display in results of each statement.\n\nDo you want to disable result set fetching for this script execution?
+sql_editor_confirm_no_fetch_result_for_big_script_yes = Yes
+sql_editor_confirm_no_fetch_result_for_big_script_no = No
+sql_editor_confirm_no_fetch_result_for_big_script_remember = Remember my choice for this editor
```

Comment:
If it saves that preference just for the current session, can we express that in the message?

Reply:
I'm able to think out something like `Remember my choice for this editor in the current session`, but it seems cumbersome for me. What do you think?

Reply:
I don't think its cumbersomness is too bad, since it explicitly tells that it does.

Maybe just rephrase it a bit:
> Don't show this message again for this editor till the end the session